tests/Oryx.BuildImage.Tests/BenvTest.cs

```diff
@@ -29,34 +29,29 @@ public BenvTest(ITestOutputHelper output)
 
         [Theory]
         // DotNet
-        [InlineData("dotnet")]
+        [InlineData("dotnet", "/opt/dotnet/")]
```

Comment:
Shouldn't we also check that those tools are in the path?

Reply:
Shouldn't this test be enough though? This test verifies that these tools are accessible out of the box.